src/components/screens/consul/Proposals/NewProposal.js

```diff
@@ -325,33 +321,78 @@ export const NewProposal = ({ navigation, data, query }) => {
   );
 };
 
-const ImageSelector = ({ control, imageUri, item, onChange, selectImage, value }) => {
-  const { buttonTitle, infoText, label, name } = item;
+const ImageSelector = ({ control, field, item, selectImage }) => {
+  const [errorMessage, setErrorMessage] = useState();
```

Comment:
we see `errorMessage` always as string, right? event if this is no typescript file, i guess having a string as initial value is more clear `useState('');`

Reply:
18e756843bc8ae851b92b31cf86be6945d2450ab